File: OpenControl/Core/Network/DTO/Auth/Responses/AuthenticationUserResponse.swift
//
//  AuthenticationUserResponse.swift
//  OpenControl
//
//  Created by Vladislav Panev on 21.05.2023.
//

import Foundation

struct AuthenticationUserResponse: Codable {
    let message: String
    let isAuthenticated: Bool
    let userName: String?
    let email: String?
    let roles: String?
    let token: String?
}

File: Back-end/Leader02.Api/Models/Response/Auth/AuthenticationUserResponse.cs
namespace Leader02.Api.Models.Response.Auth;

public class AuthenticationUserResponse
{
    public string Message { get; set; } = string.Empty;
    public bool IsAuthenticated { get; set; }
    public string? UserName { get; set; }
    public string? Email { get; set; }
    public string? Role { get; set; }
    public string? Token { get; set; }
}